pkg/ctl/gitops/apply.go

```diff
@@ -0,0 +1,178 @@
+package gitops
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/tools/clientcmd"
+
+	kubeclient "k8s.io/client-go/kubernetes"
```

Comment:
This should be part of the imports above, leading to a single group of external imports (we voted on this on Slack).